### ISSUE TITLE
Proper citation required

### ISSUE THREAD

**Issue by jrosebr1** [hritik5102/Tata_Innoverse_SolverHunt8#1]
Hi @hritik5102 -- Adrian here from PyImageSearch.com. it looks like you're using code from two of my tutorials:

1. https://www.pyimagesearch.com/2020/06/01/opencv-social-distancing-detector/
2. https://www.pyimagesearch.com/2020/05/04/covid-19-face-mask-detector-with-opencv-keras-tensorflow-and-deep-learning/

While my code and tutorials are free for you to learn from, my FAQ states that you need to cite the original tutorials if you are using them in your own projects (otherwise it is plagiarism):

https://www.pyimagesearch.com/faqs/single-faq/how-do-i-reference-or-cite-one-of-your-blog-posts-books-or-courses

Kindly update your GitHub repo and all Jupyter Notebooks you have posted on Colab to properly cite the materials. Until that is done the MIT License you have added to this repo is not valid.

Thanks!

**Comment by jrosebr1** [comment]
Hey there @hritik5102, I noticed you deleted your Facebook post about the issue but did not resolve it here. Kindly update your repo. Thanks!

**Comment by hritik5102** [comment]
Sorry for this big mistake, I literally regret for what I have done, as I always give acknowledge to each blog and material which I used in my repository. You can see my face mask custom model classification their I referred everyone. I understand how it feels to not acknowledge someone for there huge efforts. 

I promise I will never do this again in near future :( 

Following reference link added in ReadME page as well as in the code 



- [x] [Pyimagesearch - How OpenCV’s blobFromImage works](https://www.pyimagesearch.com/2017/11/06/deep-learning-opencvs-blobfromimage-works/)
- [x] [Pyimagesearch - fine tuning with keras and deep learning](https://www.pyimagesearch.com/2019/06/03/fine-tuning-with-keras-and-deep-learning/)
- [x] [PyimageSearch - find distance camera objectmarker using python opencv](https://www.pyimagesearch.com/2015/01/19/find-distance-camera-objectmarker-using-python-opencv/)

- [x] [PyimageSearch - non maximum suppression object detection python](https://www.pyimagesearch.com/2014/11/17/non-maximum-suppression-object-detection-python/)

- [x] [PyimageSearch - yolo object detection with opencv](https://www.pyimagesearch.com/2018/11/12/yolo-object-detection-with-opencv/)

- [x] [Pyimagesearch - face mask detector with opencv keras tensorflow and deep learning](https://www.pyimagesearch.com/2020/05/04/covid-19-face-mask-detector-with-opencv-keras-tensorflow-and-deep-learning/)

- [x] [Pyimagesearch - opencv social distancing detector](https://www.pyimagesearch.com/2020/06/01/opencv-social-distancing-detector/)

Check the reference.md file

**Comment by hritik5102** [comment]
If anything still remains, tell me I'll add that too in the readMe file.

**Comment by jrosebr1** [comment]
Thank you @hritik5102. I appreciate you addressing the issue and promptly ensuring it is resolved. Closing out the issue now.
